### DISCUSSION TITLE
OpenReplay

### DISCUSSION THREAD

**Discussion by SHCDevelops** [coollabsio/coolify#4504] [Service Template Requests]
It's hard enough to install OpenReplay on its own, it would be great if there was an option to do it from a template.

**Comment by SHCDevelops** [comment]
https://github.com/openreplay/openreplay

**Comment by viisrael** [comment]
Hey, @SHCDevelops did you a way to install OpenReplay on coolify?

**Comment by alepouna** [comment]
This would be nice since OpenReplay supports docker
